test/unit/transient-error-lookup-test.js

```diff
@@ -0,0 +1,20 @@
+const TransientErrorLookup = require('../../src/transient-error-lookup').TransientErrorLookup;
+
```

Comment:
btw, this test actually caught a bug when I switched to Array.indexOf per your suggestion. I had <code>return transientErrors.indexOf(error) === -1;</code> where I should have <code>return transientErrors.indexOf(error) !== -1;</code> and this test failed.